### PR TITLE
logout button inside of a dropdown in the directory app

### PR DESCRIPTION
**Pull request by bak45247**

### CODE REVIEW

File: app/templates/base.html [hunk 1]
```diff
@@ -1,87 +1,77 @@
...
-            integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
-            crossorigin="anonymous"></script>
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js"
-            integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh"
-            crossorigin="anonymous"></script>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css"
-          integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+        <!-- Bootstrap core CSS -->
+        <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
```
Comment (c-schwarze) [inline app/templates/base.html]: JS files should be defined at the end of the body.

File: app/templates/base.html [hunk 2]
```diff
@@ -1,87 +1,77 @@
...
-<style>
-    #page-content-wrapper {
-        padding: 1em;
-    }
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js"
+                integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ"
+                crossorigin="anonymous"></script>
+    </head>
+    <style>
+        #page-content-wrapper {
```
Comment (c-schwarze) [inline app/templates/base.html]: These styles should be pulled out to a directory.css file and then included here.

File: app/templates/base.html [hunk 3]
```diff
@@ -1,87 +1,77 @@
...
-                        target="_blank" rel="noopener">Need help?</a> | <a href="https://www.bethel.edu" target="_blank"
-                                                                           rel="noopener">www.bethel.edu</a><br>
-                    Copyright © {{ now.strftime('%Y') }} Bethel University
+        a {
+            color: #0069aa;
+        }
+    </style>
+    <nav class="navbar navbar-light" style="background-color: #0F2C52; max-height: 60px">
+        <a class="navbar-brand" href="/">
+            <img src="{{ url_for('static', filename='src/img/logo-directory.png') }}" height="56"
```
Comment (c-schwarze) [inline app/templates/base.html]: i'm a little nervous that this has a height="56". Does this work fine for all views? (desktop, mobile, tablet, etc.)

File: app/templates/base.html [hunk 4]
```diff
@@ -1,87 +1,77 @@
...
+                <a class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" style="color: #ffffff">
+                    <span class="u-fs14 d-none  d-sm-none  d-md-inline-block">Menu</span>
+                </a>
+                <div class="dropdown-menu dropdown-menu-right" style="right: 0; left: auto;">
+                    <a class="dropdown-item" href="{{ url_for('logout') }}">Log out</a>
+                </div>
+    </nav>
+    <div id="page-content-wrapper">
+        {% block content %}
+            <!-- Where the content of the templates go -->
```
Comment (c-schwarze) [inline app/templates/base.html]: 👍 nice

File: app/templates/base.html [hunk 5]
```diff
@@ -1,87 +1,77 @@
...
+            <span class="text-muted  u-fs14"><a href="https://www.bethel.edu/about/contact/questions-comments"
+                                                target="_blank" rel="noopener">Comments &amp; Questions</a> | <a
+                    href="https://confluence.bethel.edu/display/commmarketing/MyBethel+Documentation"
+                    target="_blank" rel="noopener">Need help?</a> | <a href="https://www.bethel.edu" target="_blank"
+                                                                       rel="noopener">www.bethel.edu</a><br>
+                Copyright © {{ now.strftime('%Y') }} Bethel University
+            </span>
+        </div>
+    </footer>
 </html>
```
Comment (c-schwarze) [inline app/templates/base.html]: These tags make me think there isn't a body tag in this file? If there isn't, you should add them

File: app/templates/base.html [hunk 6]
```diff
@@ -1,87 +1,77 @@
...
+            <!-- Where the content of the templates go -->
+        {% endblock %}
     </div>
-</footer>
-</body>
+    <footer class="footer">
+        <div class="inner">
+            <span class="text-muted  u-fs14"><a href="https://www.bethel.edu/about/contact/questions-comments"
+                                                target="_blank" rel="noopener">Comments &amp; Questions</a> | <a
+                    href="https://confluence.bethel.edu/display/commmarketing/MyBethel+Documentation"
```
Comment (c-schwarze) [inline app/templates/base.html]: This is a mybethel url

File: app/templates/base.html (hunk 3 above)
Comment (bak45247) [reply app/templates/base.html]: To be honest, a lot of this is stuff that Grant wrote before so Im not necessarily sure where a lot of it comes from. However, the height of 56 works for all different views

Comment (c-schwarze) [reply app/templates/base.html]: No worries, just wanted to check.

File: app/templates/base.html (hunk 1 above)
Comment (bak45247) [reply app/templates/base.html]: done

File: app/templates/base.html (hunk 2 above)
Comment (bak45247) [reply app/templates/base.html]: done

File: app/templates/base.html (hunk 6 above)
Comment (bak45247) [reply app/templates/base.html]: goes to the all updates page right now as we dont actually have a docu page for directory

Comment (c-schwarze) [reply app/templates/base.html]: Yeah. We could probably just get rid of that for now

File: app/templates/base.html (hunk 4 above)
Comment (bak45247) [reply app/templates/base.html]: thanks

File: app/templates/base.html (hunk 5 above)
Comment (bak45247) [reply app/templates/base.html]: done

**Pull request closed without merging**